plugins/syncthing/strelaysrv_

```diff
@@ -0,0 +1,149 @@
+#!/usr/bin/env sh
+: <<=cut
+=head1 NAME
+strelaysrv_ - Plugin to monitor Syncthing relay server
+
+=head1 DESCRIPTION
+This plugin gathers metrics from a Syncthing relay server.
+
+This plugin requires the jq utility : https://stedolan.github.io/jq/
+This plugin requires the curl utility : https://curl.haxx.se/
+
+Available plugins :
+strelaysrv_goroutine #
+strelaysrv_num       # 
+strelaysrv_proxied   #
+strelaysrv_transfer  #
+strelaysrv_uptime    #
+
+=head1 CONFIGURATION
+To make the plugin connect to the Syncthing relay server one has to use this type of 
+configuration
+[strelaysrv_*]
+
+env.syncthing_relaysrv_host 127.0.0.1
+env.syncthing_relaysrv_port 22070
+
+=head1 AUTHOR
+Pierre-Alain TORET <pierre-alain.toret@protonmail.com>
+
+=head1 LICENSE
+MIT
+=cut
+
+num() {
+    case $1 in
+        config)
+            cat <<'EOM'
+graph_title Syncthing relay numbers
+graph_category network
+graph_vlabel numbers
+strelaysrv_num_sessions.label sessions
+strelaysrv_num_connections.label connections
+strelaysrv_num_pending.label pending session keys
+strelaysrv_num_proxies.label proxies
+EOM
+            exit 0;;
+        *)
+            STATUS=$($CURL -s http://"$syncthing_relaysrv_host":"$syncthing_relaysrv_port"/status)
+            NS=$(echo "$STATUS" | $JQ '.numActiveSessions ')
+            NC=$(echo "$STATUS" | $JQ '.numConnections ')
+            NK=$(echo "$STATUS" | $JQ '.numPendingSessionKeys ')
+            NP=$(echo "$STATUS" | $JQ '.numProxies ')
+            printf "strelaysrv_num_sessions.value %s\\n" "$NS"
```

Comment:
The double backslash is not necessary (and could be wrong?).
Same goes for the next lines.